package/MDAnalysis/converters/RDKit.py

```diff
@@ -452,7 +452,7 @@ def set_converter_cache_size(maxsize):
         conversions in memory. Using ``maxsize=None`` will remove all limits
         to the cache size, i.e. everything is cached.
     """
-    global atomgroup_to_mol
+    global atomgroup_to_mol # pylint: disable=global-statement
```

Comment:
I believe we said this was a justified global call.

Reply:
I think so... caches are iffy.